variables.tf

```diff
@@ -21,15 +21,10 @@ variable "lambda_s3_bucket" {
   type        = string
 }
 
-variable "lambda_version" {
-  description = "The version the Lambda function to deploy."
+variable "lambda_package_key" {
+  description = "The object key for the lambda distribution. Defaults to lambda.zip since this is build by default upstream."
   type        = string
-}
-
-variable "lambda_package" {
-  description = "The name of the lambda package. Used for a directory tree and zip file."
-  type        = string
-  default     = "anti-virus"
+  default     = "lambda.zip"
```

Comment:
I have modified this to use the default construction if this is not given. It is a bit awkward, since you still must give a version even though it won't be used if this is given, but at least it allows it to be overwritten. I think the concept of taking a version argument at all is a bit strange, since you can pass any version here no matter what version is actually built.

Let me know if you'd like me to implement this differently. I'm not sure what you envisioned.

Reply:
@dynamike: I'd love to get your feedback on the modification made in response to your initial feedback. If not, let me know how to proceed. It look like this may conflict with implementation done by @eboboshka in #27, which is fine. I just don't want to leave it hanging until it is a distant memory.

Reply:
yes, think you for the ping. I'll review things today.